samples/BenchmarkDotNet.Samples/IntroVisualStudioDiagnoser.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using BenchmarkDotNet.Attributes;
+using Microsoft.VSDiagnostics;
+
+namespace BenchmarkDotNet.Samples
+{
+    // Enables profiling with the CPU Usage tool
+    // See: https://learn.microsoft.com/visualstudio/profiling/profiling-with-benchmark-dotnet
+    [CPUUsageDiagnoser]
```

Comment:
I gave it a try and noticed that VS was not able to solve the symbols for the samples app:

![{C5CB07C8-56C8-42FB-BDAF-65808EBB4B8D}](https://github.com/user-attachments/assets/a7284b48-fe87-4d32-bfbc-ed4b84a005e6)

I saw the names of `.pdb` it was searching for and I got to a conclusion that these were the `.pdb` of the app generated by BDN. BDN by default removes the generated app, but we can tell it to not do that.

Then everything is solved and works like a charm:

![{CE2FF81B-AD91-4C68-9B12-305A0FA7923A}](https://github.com/user-attachments/assets/b80f3862-5fe0-4669-9149-5d054da9afd8)

For now we can update the samples to include `[KeepBenchmarkFiles]` attribute or change the `VSDiagnosticsToolAttribute` attribute to do it for us: 

```diff
public abstract class VSDiagnosticsToolAttribute : Attribute, IConfigSource
{
    public string? OutputFileName;

-    public virtual IConfig Config => (IConfig)(object)ManualConfig.CreateEmpty().AddDiagnoser((IDiagnoser[])
+    public virtual IConfig Config => (IConfig)(object)ManualConfig.CreateEmpty().KeepBenchmarkFiles().AddDiagnoser((IDiagnoser[])(object)new IDiagnoser[1]
    {
        new VSDiagnosticsDiagnoser(Tool, OutputFileName, ValidateParams)
    });
```

```suggestion
    [CPUUsageDiagnoser]
    [KeepBenchmarkFiles]
```

Reply:
Oh, thanks for the heads up. Just curious, what version of VS are you on? We added something to quick cache the PDBs in the diagsession but I like your fix of just adding KeepBenchmarkFiles to the VSDiagnosticsToolAttribute. I'll get that change in real quick and push a new version of the NuGet then update this.

Reply:
I am on `Version 17.12.0 Preview 1.0 [35131.273.main]`

Reply:
Ahhh, thanks for confirming. Yeah, the caching work didn't get in until 17.13. I have a PR out to add your solution to the VSDiagnosticsToolAttribute since that will help cover more versions. Once that gets in today, I'll kick off release and update the version of the diagnosers in this PR. Thanks!

Reply:
Updated to the latest version of the diagnosers that has your fix for KeepBenchmarkFiles